src/seedu/addressbook/Main.java

```diff
@@ -1,65 +1,56 @@
 package seedu.addressbook;
 
-import seedu.addressbook.model.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.ReadOnlyPerson;
 import seedu.addressbook.storage.StorageFile.*;
 
 import seedu.addressbook.commands.*;
-import seedu.addressbook.model.AddressBook;
+import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.parser.Parser;
 import seedu.addressbook.storage.StorageFile;
 import seedu.addressbook.ui.TextUi;
 
-import java.io.*;
 import java.util.List;
 import java.util.Optional;
 
 
 /**
- * Entry point of the address book application.
+ * Entry point of the Address Book application.
  * Initializes the application and starts the interaction with the user.
  */
 public class Main {
 
-    /**
-     * Version info of the program.
-     */
+    /** Version info of the program. */
     public static final String VERSION = "AddessBook Level 2 - Version 1.0";
 
     private TextUi ui;
     private StorageFile storage;
     private AddressBook addressBook;
 
-    /**
-     * The list of person shown to the user most recently.
-     */
+    /** The list of person shown to the user most recently.  */
     private List<? extends ReadOnlyPerson> lastShownList = null;
 
 
     public static void main(String... launchArgs) {
-        new Main().run(launchArgs, System.in, System.out);
+        new Main().run(launchArgs);
```

Comment:
I guess you added these two parameters to use Junit for I/O testing? If yes, we'll bring them back later when we need it.